cli/cmd/release_lint.go

```diff
@@ -45,7 +47,15 @@ func (r *runners) releaseLint(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		r.args.lintReleaseYamlFile = string(bytes)
+		r.args.lintReleaseYaml = string(bytes)
```

Comment:
This was a bug, couldn't lint ship releases from CLI with the `--yaml-file` flag